2-Browsers/Week1/homework/ex1-bookList/index.js

```diff
@@ -1,24 +1,23 @@
-//cspell: disable
-/*------------------------------------------------------------------------------
-Full description at: https://github.com/HackYourFuture/Homework/tree/main/2-Browsers/Week1#exercise-1-the-book-list
-
-I'd like to display my three favorite books inside a nice webpage!
-
-1. Iterate through the array of books.
-2. For each book, create a `<p>`
-element with the book title and author.
-3. Use a `<ul>`  and `<li>` to display the books.
-4. Add an `<img>` to each book that links to a URL of the book cover.
-5. Change the style of the book depending on whether you have read it(green) or not(red).
-
-The end result should look something like this:
-https://hackyourfuture.github.io/example-pages/Browsers/Week1/1-booklist/
-
------------------------------------------------------------------------------*/
-//cspell: enable
-
 function createBookList(books) {
-  // TODO your code goes in here, return the ul element
+  const ul = document.createElement('ul');
+  for (let i = 0; i < books.length; i++) {
+    const li = document.createElement('li');
+    const paragraph = document.createElement('p');
+    const img = document.createElement('img');
+    paragraph.textContent = ' ${books[i].title} by ${books[i].author}';
```

Comment:
Hi @chouaib-at , just one more comment here: template literals with string interpolation use "backticks" (see documentation here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals). If you use ' ' then it will not work (see screenshot)

```suggestion
    paragraph.textContent = `${books[i].title} by ${books[i].author}`;
```

<img width="910" alt="Screen Shot 2023-06-26 at 15 53 24" src="https://github.com/HackYourHomework/Homework-class44/assets/23385221/1408f89b-d75f-4799-99d2-80e489e23f28">